packages/signature-controller/src/SignatureController.test.ts

```diff
@@ -521,10 +519,10 @@ describe('SignatureController', () => {
     it('throws if approval rejected', async () => {
       messengerMock.call
         .mockResolvedValueOnce({}) // LoggerController:add
-        .mockRejectedValueOnce({}); // ApprovalController:addRequest
+        .mockRejectedValueOnce({ message: 'User rejected the request.' }); // ApprovalController:addRequest
```

Comment:
since we are mocking the request, how do we ensure 'User rejected the request.' is still being thrown?

Reply:
We only ensure whatever error was thrown by the client is returned as is. It is the responsibility of the client/UI in this case to throw the right error.